git_machete/client.py

```diff
@@ -2086,7 +2086,15 @@ def __get_remote_and_organization_and_repository_name_for_custom_url(self) -> Re
                                                   repository=self.__git.get_config_attr_or_none("machete.github.repository"))
 
     def create_github_pr(
-            self, *, head: LocalBranchShortName, opt_draft: bool, opt_onto: Optional[LocalBranchShortName]) -> None:
+            self,
+            *,
```

Comment:
Huh I've already forgotten... what's this `*` for in this method? 🤔

Reply:
@PawelLipski Parameters after `*` are keyword-only parameters and may only be passed used keyword arguments. I would say the only use case i see is when you change order of the arguments, it protects you from passing values to wrong arguments, not sure if this is really necessary here since the function is only used once and even if you don't put '*' in the function def you can still pass parameters using keyword-args, but overall pretty interesting feature.

Reply:
Oh! okay, makes sense, let's leave as is, indeed such extra protection is useful (could even be applied in more places TBH... I'll open an issue)

Reply:
https://github.com/VirtusLab/git-machete/issues/568